subworkflows/nf-scil/preproc_dwi/meta.yml

```diff
@@ -0,0 +1,113 @@
+name: "preproc_dwi"
```

Comment:
Here, in the description of the steps, keep an imperative verb tense, at least for the first sentence where you describe the behavior. Corrects for ..., Used to remove ..., Extracts the mean ... . Then you can switch when you describe further and it matches.